src/binding.cc

```diff
@@ -346,7 +378,7 @@ char** deno_argv() { return global_argv; }
 int deno_argc() { return global_argc; }
 
 void deno_set_flags(int* argc, char** argv) {
-  v8::V8::SetFlagsFromCommandLine(argc, argv, true);
+  // v8::V8::SetFlagsFromCommandLine(argc, argv, true);
```

Comment:
If you're doing this to get access to gtest command line flags, it's better we just remove the `deno_set_flags` in `src/tests.cc`.